### PR TITLE
looks good

### PR DESCRIPTION
**Pull request by Havrysh**

### CODE REVIEW

**Comment by Havrysh** [comment]
https://havrysh.github.io/react_list-of-goods/

File: src/App.tsx [hunk 1]
```diff
@@ -15,33 +16,68 @@ const goodsFromServer: string[] = [
...
+            </button>
+
+            <button type="button" onClick={() => setOrder(true)}>
+              Reset
+            </button>
+
+            {/* Can't sort or reverse the list items after I pressed reset button. Maybe it should be this way if no - let me know please */}
+            {!initialOrder
+              ? <GoodsList goods={visibleGoods} />
+              : <GoodsList goods={goodsFromServer} />}
```
Comment (ArturErmolenko) [inline src/App.tsx:76]: nope it shouldn't be this way, need to fix this

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (Havrysh) [reply src/App.tsx:76]: Share the idea

Comment (ArturErmolenko) [reply src/App.tsx:76]: just create handlers and pass them to buttons, and when those button are clicked – perform the action (i.e. sort, reset, show – and so on)

**Review by ArturErmolenko — APPROVED** [review]

**Pull request closed without merging**